lib/workers/repository/update/pr/code-owners.spec.ts

```diff
@@ -22,13 +22,142 @@ describe('workers/repository/update/pr/code-owners', () => {
       expect(codeOwners).toEqual(['@jimmy']);
     });
 
+    it('respects orphan files', async () => {
+      fs.readLocalFile.mockResolvedValueOnce(
+        ['* @jimmy', 'yarn.lock'].join('\n')
+      );
+      git.getBranchFiles.mockResolvedValueOnce(['yarn.lock']);
+      const codeOwners = await codeOwnersForPr(pr);
+      expect(codeOwners).toEqual([]);
+    });
+
     it('returns more specific code owners', async () => {
       fs.readLocalFile.mockResolvedValueOnce(
         ['* @jimmy', 'package.json @john @maria'].join('\n')
       );
       git.getBranchFiles.mockResolvedValueOnce(['package.json']);
       const codeOwners = await codeOwnersForPr(pr);
-      expect(codeOwners).toEqual(['@john', '@maria']);
+      expect(codeOwners).toEqual(['@john', '@maria', '@jimmy']);
+    });
+
+    describe('returns more specific code owners in monorepos', () => {
+      const mockCodeOwners = `
```

Comment:
```suggestion
      const mockCodeOwners = codeBlock`
```
use template helper, which strips leading whitespace to ensure proper alignment